GatewayIntegrationTest.shouldReturnRejectionWithCorrectTypeAndReason flaky

**Summary**

- How often does the test fail? At least once so far
- Does it block your work? no
- Do we suspect that it is a real failure? unknown

**Failures**

<details><summary>Example assertion failure</summary>
<pre>
java.lang.AssertionError: 

Expecting:
  <java.util.concurrent.TimeoutException: Request timed out after PT3S>
to be an instance of:
  <io.zeebe.gateway.cmd.BrokerRejectionException>
but was:
  <"java.util.concurrent.TimeoutException: Request timed out after PT3S
	at io.zeebe.transport.impl.RequestContext.timeout(RequestContext.java:112)
	at io.zeebe.transport.impl.AtomixClientTransportAdapter.timeoutFuture(AtomixClientTransportAdapter.java:160)
	at io.zeebe.transport.impl.AtomixClientTransportAdapter.lambda$sendRequestInternal$1(AtomixClientTransportAdapter.java:81)
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76)
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39)
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:118)
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:107)
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:91)
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:204)
">
	at io.zeebe.broker.it.gateway.GatewayIntegrationTest.shouldReturnRejectionWithCorrectTypeAndReason(GatewayIntegrationTest.java:82)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:258)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeLazy(JUnitCoreWrapper.java:119)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:87)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:157)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
</pre>
</details>

**Hypotheses**


**Logs**

<details><summary>Logs</summary>
<pre>
14:45:48.417 [] [main] INFO  io.zeebe.test - Test started: shouldReturnRejectionWithCorrectTypeAndReason(io.zeebe.broker.it.gateway.GatewayIntegrationTest)
14:45:48.805 [] [main] INFO  io.zeebe.test.util.SocketUtil - Starting socket assignment with testForkNumber 6 and testMavenId 2
14:45:48.917 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 31800 for test fork 6 with range PortRange{host='localhost', basePort=31800, maxOffset=100, currentOffset=1}
14:45:48.923 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 31801 for test fork 6 with range PortRange{host='localhost', basePort=31800, maxOffset=100, currentOffset=2}
14:45:48.925 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 31802 for test fork 6 with range PortRange{host='localhost', basePort=31800, maxOffset=100, currentOffset=3}
14:45:48.927 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 31803 for test fork 6 with range PortRange{host='localhost', basePort=31800, maxOffset=100, currentOffset=4}
14:45:48.928 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 31804 for test fork 6 with range PortRange{host='localhost', basePort=31800, maxOffset=100, currentOffset=5}
14:45:49.025 [] [main] DEBUG io.zeebe.broker.system - Initializing system with base path /tmp/97af0648-ba7e-40ac-9593-9d2ae8985d42
14:45:49.522 [] [main] INFO  io.zeebe.broker.system - Version: 0.25.0-SNAPSHOT
14:45:52.129 [] [main] INFO  io.zeebe.broker.system - Starting broker 0 with configuration {
  "network" : {
    "host" : "0.0.0.0",
    "portOffset" : 0,
    "maxMessageSize" : "4MB",
    "advertisedHost" : "0.0.0.0",
    "commandApi" : {
      "host" : "0.0.0.0",
      "port" : 31802,
      "advertisedHost" : "0.0.0.0",
      "advertisedPort" : 31802,
      "address" : "0.0.0.0:31802",
      "advertisedAddress" : "0.0.0.0:31802"
    },
    "internalApi" : {
      "host" : "0.0.0.0",
      "port" : 31803,
      "advertisedHost" : "0.0.0.0",
      "advertisedPort" : 31803,
      "address" : "0.0.0.0:31803",
      "advertisedAddress" : "0.0.0.0:31803"
    },
    "monitoringApi" : {
      "host" : "0.0.0.0",
      "port" : 31804,
      "advertisedHost" : "0.0.0.0",
      "advertisedPort" : 31804,
      "address" : "0.0.0.0:31804",
      "advertisedAddress" : "0.0.0.0:31804"
    },
    "maxMessageSizeInBytes" : 4194304
  },
  "cluster" : {
    "initialContactPoints" : [ ],
    "partitionIds" : [ 1 ],
    "nodeId" : 0,
    "partitionsCount" : 1,
    "replicationFactor" : 1,
    "clusterSize" : 1,
    "clusterName" : "zeebe-cluster",
    "membership" : {
      "broadcastUpdates" : false,
      "broadcastDisputes" : true,
      "notifySuspect" : false,
      "gossipInterval" : "PT0.25S",
      "gossipFanout" : 2,
      "probeInterval" : "PT1S",
      "probeTimeout" : "PT2S",
      "suspectProbes" : 3,
      "failureTimeout" : "PT10S",
      "syncInterval" : "PT10S"
    }
  },
  "threads" : {
    "cpuThreadCount" : 2,
    "ioThreadCount" : 2
  },
  "data" : {
    "directories" : [ "/tmp/97af0648-ba7e-40ac-9593-9d2ae8985d42/data" ],
    "logSegmentSize" : "512MB",
    "snapshotPeriod" : "PT15M",
    "logIndexDensity" : 100,
    "diskUsageReplicationWatermark" : 0.9,
    "diskUsageCommandWatermark" : 0.8,
    "diskUsageMonitoringInterval" : "PT1S",
    "logSegmentSizeInBytes" : 536870912,
    "atomixStorageLevel" : "DISK",
    "freeDiskSpaceCommandWatermark" : 0,
    "freeDiskSpaceReplicationWatermark" : 0
  },
  "exporters" : {
    "test-recorder" : {
      "jarPath" : null,
      "className" : "io.zeebe.test.util.record.RecordingExporter",
      "args" : null,
      "external" : false
    }
  },
  "gateway" : {
    "network" : {
      "host" : "0.0.0.0",
      "port" : 31800,
      "minKeepAliveInterval" : "PT30S"
    },
    "cluster" : {
      "contactPoint" : "0.0.0.0:31803",
      "requestTimeout" : "PT15S",
      "clusterName" : "zeebe-cluster",
      "memberId" : "gateway",
      "host" : "0.0.0.0",
      "port" : 31801,
      "membership" : {
        "broadcastUpdates" : false,
        "broadcastDisputes" : true,
        "notifySuspect" : false,
        "gossipInterval" : "PT0.25S",
        "gossipFanout" : 2,
        "probeInterval" : "PT1S",
        "probeTimeout" : "PT2S",
        "suspectProbes" : 3,
        "failureTimeout" : "PT10S",
        "syncInterval" : "PT10S"
      }
    },
    "threads" : {
      "managementThreads" : 1
    },
    "monitoring" : {
      "enabled" : false,
      "host" : "0.0.0.0",
      "port" : 9600
    },
    "security" : {
      "enabled" : false,
      "certificateChainPath" : null,
      "privateKeyPath" : null
    },
    "longPolling" : {
      "enabled" : true
    },
    "initialized" : true,
    "enable" : false
  },
  "backpressure" : {
    "enabled" : true,
    "algorithm" : "VEGAS",
    "aimd" : {
      "requestTimeout" : "PT1S",
      "initialLimit" : 100,
      "minLimit" : 1,
      "maxLimit" : 1000,
      "backoffRatio" : 0.9
    },
    "fixedLimit" : {
      "limit" : 20
    },
    "vegas" : {
      "alpha" : 3,
      "beta" : 6,
      "initialLimit" : 20
    },
    "gradient" : {
      "minLimit" : 10,
      "initialLimit" : 20,
      "rttTolerance" : 2.0
    },
    "gradient2" : {
      "minLimit" : 10,
      "initialLimit" : 20,
      "rttTolerance" : 2.0,
      "longWindow" : 600
    }
  },
  "stepTimeout" : "PT5M",
  "executionMetricsExporterEnabled" : false
}
14:45:52.407 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [1/12]: actor scheduler
14:45:52.703 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [1/12]: actor scheduler started in 295 ms
14:45:52.704 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [2/12]: membership and replication protocol
14:46:10.109 [] [main] DEBUG io.atomix.utils.serializer.Namespace - (no name): class io.atomix.utils.time.LogicalTimestamp already registered as 53. Skipping 58.
14:46:10.110 [] [main] DEBUG io.atomix.utils.serializer.Namespace - (no name): class io.atomix.utils.time.WallClockTimestamp already registered as 54. Skipping 59.
14:46:10.128 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [2/12]: membership and replication protocol started in 17424 ms
14:46:10.129 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [3/12]: command api transport
14:46:11.024 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:31802
14:46:11.042 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
14:46:11.043 [] [main] DEBUG io.zeebe.broker.system - Bound command API to 0.0.0.0:31802 
14:46:11.112 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [3/12]: command api transport started in 982 ms
14:46:11.113 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [4/12]: command api handler
14:46:11.313 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [4/12]: command api handler started in 200 ms
14:46:11.313 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [5/12]: subscription api
14:46:11.530 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [5/12]: subscription api started in 216 ms
14:46:11.531 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [6/12]: cluster services
14:46:11.533 [] [main] INFO  io.atomix.core.Atomix - 0.25.0-SNAPSHOT (revision 9049b0 built on 2020-07-14 14:42:47)
14:46:11.616 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:31803
14:46:11.618 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
14:46:11.807 [] [netty-unicast-event-nio-client-0] INFO  io.atomix.cluster.messaging.impl.NettyUnicastService - UDP server listening for connections on 0.0.0.0:31803
14:46:11.810 [] [atomix-cluster-0] INFO  io.atomix.cluster.discovery.BootstrapDiscoveryProvider - Joined
14:46:11.815 [] [atomix-cluster-0] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - Nodes from discovery service []
14:46:11.822 [] [atomix-cluster-0] INFO  io.atomix.cluster.protocol.SwimMembershipProtocol - Started
14:46:11.823 [] [atomix-cluster-0] INFO  io.atomix.cluster.impl.DefaultClusterMembershipService - Started
14:46:11.823 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterCommunicationService - Started
14:46:11.825 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterEventService - Started
14:46:11.838 [] [atomix-0] INFO  io.atomix.raft.partition.impl.RaftPartitionServer - Starting server for partition PartitionId{id=1, group=raft-partition}
14:46:12.010 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 500.
14:46:12.011 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 501.
14:46:12.011 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 502.
14:46:12.012 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 503.
14:46:12.013 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 504.
14:46:12.014 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 505.
14:46:12.014 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 506.
14:46:12.014 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 507.
14:46:12.014 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 508.
14:46:12.017 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 509.
14:46:12.017 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 510.
14:46:12.017 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 511.
14:46:12.017 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 512.
14:46:12.017 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 513.
14:46:12.025 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 530.
14:46:12.025 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 531.
14:46:12.035 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 535.
14:46:12.035 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 536.
14:46:12.036 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 537.
14:46:12.036 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 538.
14:46:12.037 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 541.
14:46:12.039 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 542.
14:46:12.040 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 543.
14:46:12.040 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 544.
14:46:12.040 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 545.
14:46:12.040 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 546.
14:46:12.041 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 547.
14:46:12.041 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 548.
14:46:12.042 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 549.
14:46:12.042 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 550.
14:46:12.042 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: class java.util.ArrayList already registered as 27. Skipping 551.
14:46:12.042 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: class java.util.LinkedList already registered as 28. Skipping 552.
14:46:12.043 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: class java.util.Collections$EmptyList already registered as 41. Skipping 553.
14:46:12.043 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: class java.util.HashSet already registered as 29. Skipping 554.
14:46:12.046 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftProtocol: void already registered as 9. Skipping 557.
14:46:12.050 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 600.
14:46:12.050 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 601.
14:46:12.051 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 604.
14:46:12.051 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 605.
14:46:12.052 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 606.
14:46:12.052 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 607.
14:46:12.052 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 608.
14:46:12.053 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 609.
14:46:12.053 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 610.
14:46:12.053 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: void already registered as 9. Skipping 611.
14:46:12.053 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: class java.util.ArrayList already registered as 27. Skipping 612.
14:46:12.053 [] [atomix-0] DEBUG io.atomix.utils.serializer.Namespace - RaftStorage: class java.util.HashSet already registered as 29. Skipping 613.
14:46:12.252 [] [atomix-0] DEBUG io.atomix.storage.journal.SegmentedJournal - Created segment: JournalSegment{id=1, version=1, index=1}
14:46:12.449 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Transitioning to FOLLOWER
14:46:12.514 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.roles.FollowerRole - RaftServer{raft-partition-partition-1}{role=FOLLOWER} - Single member cluster. Transitioning directly to candidate.
14:46:12.514 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Transitioning to CANDIDATE
14:46:12.515 [] [raft-server-0-raft-partition-partition-1] WARN  io.atomix.utils.event.ListenerRegistry - Listener io.atomix.raft.roles.FollowerRole$$Lambda$312/0x00000008403b7040@1cdf777e not registered
14:46:12.525 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.roles.CandidateRole - RaftServer{raft-partition-partition-1}{role=CANDIDATE} - Single member cluster. Transitioning directly to leader.
14:46:12.604 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Set term 1
14:46:12.612 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Voted for 0
14:46:12.613 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Transitioning to LEADER
14:46:12.631 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Found leader 0
14:46:12.843 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.impl.DefaultRaftServer - RaftServer{raft-partition-partition-1} - Server started successfully!
14:46:12.846 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.partition.impl.RaftPartitionServer - Successfully started server for partition PartitionId{id=1, group=raft-partition}
14:46:12.846 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.partition.RaftPartitionGroup - Started
14:46:12.847 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.primitive.partition.impl.DefaultPartitionService - Started class io.atomix.primitive.partition.impl.DefaultPartitionService
14:46:12.849 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [6/12]: cluster services started in 1316 ms
14:46:12.850 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [7/12]: topology manager
14:46:12.908 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [7/12]: topology manager started in 58 ms
14:46:12.908 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [8/12]: monitoring services
14:46:12.944 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [8/12]: monitoring services started in 35 ms
14:46:12.944 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [9/12]: disk space monitor
14:46:12.948 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [9/12]: disk space monitor started in 4 ms
14:46:12.948 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [10/12]: leader management request handler
14:46:13.011 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [10/12]: leader management request handler started in 63 ms
14:46:13.012 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [11/12]: zeebe partitions
14:46:13.018 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 partitions [1/1]: partition 1
14:46:13.152 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - 0 - Detected local properties change {brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDozMTgwMgUAAAwAAA8AAAAwLjI1LjAtU05BUFNIT1Q=}
14:46:13.225 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Removing follower partition service for partition PartitionId{id=1, group=raft-partition}
14:46:13.230 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Partition role transitioning from null to LEADER
14:46:13.231 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Installing leader partition service for partition PartitionId{id=1, group=raft-partition}
14:46:17.642 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.logstreams.snapshot - Opened database from '/tmp/97af0648-ba7e-40ac-9593-9d2ae8985d42/data/raft-partition/partitions/1/runtime'.
14:46:18.418 [Broker-0-LogStream-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.logstreams - Configured log appender back pressure at partition 1 as AppenderVegasCfg{initialLimit=1024, maxConcurrency=32768, alphaLimit=0.7, betaLimit=0.95}. Window limiting is disabled
14:46:18.447 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.logstreams - Recovering state of partition 1 from snapshot
14:46:19.517 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-0] INFO  org.camunda.feel.FeelEngine - Engine created. [value-mapper: CompositeValueMapper(List(io.zeebe.el.impl.feel.MessagePackValueMapper@298f86fb)), function-provider: io.zeebe.el.impl.feel.FeelFunctionProvider@75d9d333, configuration: Configuration(false)]
14:46:21.132 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-0] INFO  io.zeebe.logstreams - Recovered state of partition 1 from snapshot at position -1
14:46:21.246 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-0] INFO  org.camunda.feel.FeelEngine - Engine created. [value-mapper: CompositeValueMapper(List(io.zeebe.el.impl.feel.MessagePackValueMapper@567ee6ea)), function-provider: io.zeebe.el.impl.feel.FeelFunctionProvider@339427e7, configuration: Configuration(false)]
14:46:21.333 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-0] INFO  org.camunda.feel.FeelEngine - Engine created. [value-mapper: CompositeValueMapper(List(io.zeebe.el.impl.feel.MessagePackValueMapper@7efc014e)), function-provider: io.zeebe.el.impl.feel.FeelFunctionProvider@372054c1, configuration: Configuration(false)]
14:46:22.014 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-0] DEBUG io.zeebe.broker.exporter - Recovering exporter from snapshot
14:46:22.037 [Broker-0-HealthCheckService] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - All partitions are installed. Broker is ready!
14:46:22.110 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - 0 - Detected local properties change {brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDozMTgwMgUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}
14:46:22.116 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 partitions [1/1]: partition 1 started in 9098 ms
14:46:22.116 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 partitions succeeded. Started 1 steps in 9098 ms.
14:46:22.117 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [11/12]: zeebe partitions started in 9104 ms
14:46:22.117 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [12/12]: register diskspace usage listeners
14:46:22.118 [] [main] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [12/12]: register diskspace usage listeners started in 0 ms
14:46:22.118 [] [main] INFO  io.zeebe.broker.system - Bootstrap Broker-0 succeeded. Started 12 steps in 29711 ms.
14:46:22.118 [] [main] INFO  io.zeebe.broker.test - 
====
Broker startup time: 33412
====

14:46:22.121 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 31805 for test fork 6 with range PortRange{host='localhost', basePort=31800, maxOffset=100, currentOffset=6}
14:46:22.224 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-0] DEBUG io.zeebe.broker.exporter - Recovered exporter 'Broker-0-Exporter-1' from snapshot at lastExportedPosition -1
14:46:22.224 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-0] DEBUG io.zeebe.broker.exporter - Configure exporter with id 'test-recorder'
14:46:22.541 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-0] DEBUG io.zeebe.broker.exporter - Set event filter for exporters: ExporterEventFilter{acceptRecordTypes={COMMAND_REJECTION=true, SBE_UNKNOWN=true, COMMAND=true, EVENT=true, NULL_VAL=true}, acceptValueTypes={VARIABLE=true, JOB_BATCH=true, WORKFLOW_INSTANCE=true, WORKFLOW_INSTANCE_RESULT=true, MESSAGE=true, JOB=true, SBE_UNKNOWN=true, TIMER=true, ERROR=true, MESSAGE_START_EVENT_SUBSCRIPTION=true, INCIDENT=true, MESSAGE_SUBSCRIPTION=true, NULL_VAL=true, WORKFLOW_INSTANCE_SUBSCRIPTION=true, WORKFLOW_INSTANCE_CREATION=true, DEPLOYMENT=true, VARIABLE_DOCUMENT=true}}
14:46:22.618 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-0] DEBUG io.zeebe.broker.exporter - Open exporter with id 'test-recorder'
14:46:22.747 [GatewayTopologyManager] [gateway-scheduler-zb-actors-0] DEBUG io.zeebe.gateway - Received new broker BrokerInfo{nodeId=0, partitionsCount=1, clusterSize=1, replicationFactor=1, partitionRoles={1=LEADER}, partitionLeaderTerms={1=1}, version=0.25.0-SNAPSHOT}.
14:46:26.305 [] [main] DEBUG io.zeebe.gateway - Closing gateway broker client ...
14:46:26.308 [] [main] DEBUG io.zeebe.gateway - topology manager closed
14:46:26.310 [] [main] DEBUG io.zeebe.gateway - Gateway broker client closed.
14:46:26.322 [] [main] INFO  io.zeebe.test.records - Test failed, following records were exported:
14:46:26.330 [] [main] INFO  io.zeebe.broker.test - Test execution time: 4209
14:46:26.332 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [1/12]: register diskspace usage listeners
14:46:26.332 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [1/12]: register diskspace usage listeners closed in 0 ms
14:46:26.333 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [2/12]: zeebe partitions
14:46:26.333 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 partitions [1/1]: partition 1
14:46:26.337 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-0] DEBUG io.zeebe.broker.exporter - Closed exporter director 'Broker-0-Exporter-1'.
14:46:26.402 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.exporter.stream.ExporterDirector@1e86f4f7 successfully
14:46:26.403 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$589/0x0000000840523440@2ffde12a successfully
14:46:26.404 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.impl.AsyncSnapshotDirector@329abb02 successfully
14:46:26.406 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$376/0x00000008403cc440@4e181afa successfully
14:46:26.409 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.logstreams - Closed stream processor controller Broker-0-StreamProcessor-1.
14:46:26.410 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.engine.processing.streamprocessor.StreamProcessor@4827abde successfully
14:46:26.411 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$388/0x00000008403f9840@7e347615 successfully
14:46:26.411 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.logstreams.LogDeletionService@4776a584 successfully
14:46:26.526 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.logstreams.snapshot - Closed database from '/tmp/97af0648-ba7e-40ac-9593-9d2ae8985d42/data/raft-partition/partitions/1/runtime'.
14:46:26.528 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$386/0x00000008403f9040@5c1a4583 successfully
14:46:26.528 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$378/0x00000008403cbc40@2eb28101 successfully
14:46:26.529 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$377/0x00000008403cb840@236b03e8 successfully
14:46:26.530 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$376/0x00000008403cc440@3acaa0fa successfully
14:46:26.531 [Broker-0-LogStream-1] [Broker-0-zb-actors-0] INFO  io.zeebe.logstreams - Close appender for log stream logstream-raft-partition-partition-1
14:46:26.606 [logstream-raft-partition-partition-1-write-buffer] [Broker-0-zb-actors-0] DEBUG io.zeebe.dispatcher - Dispatcher closed
14:46:26.607 [Broker-0-LogStream-1] [Broker-0-zb-actors-0] INFO  io.zeebe.logstreams - On closing logstream logstream-raft-partition-partition-1 close 3 readers
14:46:26.608 [Broker-0-LogStream-1] [Broker-0-zb-actors-0] INFO  io.zeebe.logstreams - Close log storage with name logstream-raft-partition-partition-1
14:46:26.608 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$374/0x00000008403ccc40@4dacaa49 successfully
14:46:26.609 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 partitions [1/1]: partition 1 closed in 275 ms
14:46:26.609 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 partitions succeeded. Closed 1 steps in 276 ms.
14:46:26.609 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [2/12]: zeebe partitions closed in 276 ms
14:46:26.609 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [3/12]: leader management request handler
14:46:26.610 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [3/12]: leader management request handler closed in 0 ms
14:46:26.610 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [4/12]: disk space monitor
14:46:26.610 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [4/12]: disk space monitor closed in 0 ms
14:46:26.610 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [5/12]: monitoring services
14:46:26.611 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [5/12]: monitoring services closed in 1 ms
14:46:26.611 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [6/12]: topology manager
14:46:26.611 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [6/12]: topology manager closed in 0 ms
14:46:26.611 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [7/12]: cluster services
14:46:26.611 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [7/12]: cluster services closed in 0 ms
14:46:26.611 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [8/12]: subscription api
14:46:26.612 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [8/12]: subscription api closed in 0 ms
14:46:26.612 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [9/12]: command api handler
14:46:26.616 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [9/12]: command api handler closed in 4 ms
14:46:26.616 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [10/12]: command api transport
14:46:29.013 [] [ForkJoinPool.commonPool-worker-3] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Stopped
14:46:29.014 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [10/12]: command api transport closed in 2398 ms
14:46:29.014 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [11/12]: membership and replication protocol
14:46:29.018 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Transitioning to INACTIVE
14:46:30.904 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.impl.zeebe.ZeebeRaftStateMachine - ZeebeRaftStateMachine{raft-partition-partition-1} - Closing state machine raft-partition-partition-1
14:46:30.909 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.storage.journal.SegmentedJournal - Closing segment: JournalSegment{id=1, version=1, index=1}
14:46:31.239 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.partition.RaftPartitionGroup - Stopped
14:46:31.240 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.primitive.partition.impl.DefaultPartitionService - Stopped
14:46:31.241 [] [atomix-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterCommunicationService - Stopped
14:46:31.302 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterEventService - Stopped
14:46:31.308 [] [atomix-cluster-0] INFO  io.atomix.cluster.protocol.SwimMembershipProtocol - 0 - Member deactivated: Member{id=0, address=0.0.0.0:31803, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDozMTgwMgUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}}
14:46:31.308 [] [atomix-cluster-0] INFO  io.atomix.cluster.protocol.SwimMembershipProtocol - Stopped
14:46:31.309 [] [atomix-cluster-0] INFO  io.atomix.cluster.discovery.BootstrapDiscoveryProvider - Left
14:46:31.309 [] [atomix-cluster-0] INFO  io.atomix.cluster.impl.DefaultClusterMembershipService - Stopped
14:46:35.535 [] [ForkJoinPool.commonPool-worker-3] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Stopped
14:46:35.536 [] [atomix-cluster-0] INFO  io.atomix.cluster.AtomixCluster - Stopped
14:46:35.536 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [11/12]: membership and replication protocol closed in 6521 ms
14:46:35.537 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [12/12]: actor scheduler
14:46:35.606 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [12/12]: actor scheduler closed in 68 ms
14:46:35.606 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 succeeded. Closed 12 steps in 9275 ms.
14:46:35.606 [] [main] INFO  io.zeebe.broker.system - Broker shut down.
14:46:36.331 [] [main] INFO  io.zeebe.broker.test - Broker closing time: 9999
</pre>
</details>

--- a/qa/integration-tests/src/test/java/io/zeebe/broker/it/gateway/GatewayIntegrationTest.java
+++ b/qa/integration-tests/src/test/java/io/zeebe/broker/it/gateway/GatewayIntegrationTest.java
@@ -48,7 +48,7 @@ public final class GatewayIntegrationTest {
         .setHost("0.0.0.0")
         .setPort(SocketUtil.getNextAddress().getPort())
         .setContactPoint(internalApi.toString())
-        .setRequestTimeout(Duration.ofSeconds(3));
+        .setRequestTimeout(Duration.ofSeconds(10));
     configuration.init();
 
     final ControlledActorClock clock = new ControlledActorClock();

